Userland/Libraries/LibGfx/ImageFormats/JPEGLoader.cpp

```diff
@@ -296,11 +296,8 @@ class HuffmanStream {
     {
         m_bit_offset += count;
 
-        if (m_bit_offset > bits_in_reservoir) {
-            // FIXME: I can't find a test case for that so let's leave it for later
-            //        instead of inserting an hard-to-find bug.
-            TODO();
-        }
+        if (m_bit_offset > bits_in_reservoir)
+            TRY(refill_reservoir());
```

Comment:
I initially wrote something for this case but discarded it when I discovered that this was not covered by my test cases. And I'm sure it was longer than that, but after a day I think that I finally remember why, I had a unique logic to handle the case where `m_bit_offset` was bigger than the reservoir itself. But `refill_reservoir()` seems to handle that just fine. So, I guess than one of my early version didn't.